### DISCUSSION TITLE
Execute Web3 transactions on server side

### DISCUSSION THREAD

**Discussion by 1M4nt0** [wevm/viem#531] [Question]
At the moment, is it possibile with viem (+ wagmi) to sign a specific transaction (execution of a smart contract function) in a nextjs frontend and then send it to a backend API that forwards the transaction to the blockchain?
I need this to be sure that some tasks are executed when a smart contract function is executed.
With old @wagmi/core I encountered errors with Metamask telling me that this type of operation is unsupported cause nonce unsync.
Thank you.

**Comment by ArmmGh** [comment]
Hi, any update on this please?

**Reply by saqlain1020** [reply]
Metamask doesn't allow transaction signing so that won't work.
